cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -136,6 +137,22 @@ def _validate(self, os, scheduler):
             )
 
 
+class PrivateOsCustomAmiValidator(Validator):
+    """If OS is one of the OSSes that we do not publicly publish/release we need to use a Custom AMI."""
```

Comment:
I'd write: "For some OSes we don't publish official AMIs, so CustomAmi parameter is required"